### ISSUE TITLE
undefined method `integrate_targets'

### ISSUE THREAD

**Issue by jakedunc** [segiddins/cocoapods-dependencies#15]
I get the following every time I run `pod dependencies`, any ideas? I am on the latest cocoapods gem and I have also tried the pre version. Same error..

```
NoMethodError - undefined method `integrate_targets' for #<Pod::Config:0x007fc859328a60>
/Users/jakedunc/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/cocoapods-dependencies-0.5.2/lib/pod/command/dependencies.rb:69:in `dependencies'
/Users/jakedunc/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/cocoapods-dependencies-0.5.2/lib/pod/command/dependencies.rb:162:in `block in yaml_output'
/Users/jakedunc/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/cocoapods-1.0.1/lib/cocoapods/user_interface.rb:115:in `title'
/Users/jakedunc/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/cocoapods-dependencies-0.5.2/lib/pod/command/dependencies.rb:161:in `yaml_output'
/Users/jakedunc/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/cocoapods-dependencies-0.5.2/lib/pod/command/dependencies.rb:58:in `run'
/Users/jakedunc/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/claide-1.0.0/lib/claide/command.rb:334:in `run'
/Users/jakedunc/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/cocoapods-1.0.1/lib/cocoapods/command.rb:50:in `run'
/Users/jakedunc/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/cocoapods-1.0.1/bin/pod:55:in `<top (required)>'
/Users/jakedunc/.rbenv/versions/2.2.3/bin/pod:23:in `load'
/Users/jakedunc/.rbenv/versions/2.2.3/bin/pod:23:in `<main>'
```

**Comment by segiddins** [comment]
I can't remember updating this gem for 1.0, so that's not totally surprising. A PR fixing the issue would be appreciated, thanks!

**Comment by dnkoutso** [comment]
This appears to be fixed in `1.0.0.beta.1`.

Should there be an official 1.0.0 release?

**Comment by segiddins** [comment]
Released as 1.0.0, thanks for the reminder @dnkoutso !
